requirements.txt

```diff
@@ -27,8 +27,8 @@ einops
 flash-attn
 auto-gptq
 fire
-# alpaca-farm eval - my fork with some fixes.
-git+https://github.com/hamishivi/alpaca_farm.git
+alpaca-eval==0.3.1
```

Comment:
The one thing (discussed elsewhere) is that alpacaeval uses a different prompt to alpacafarm. I'm happy to shift to the new package, but it makes our old numbers not really comparable.